serverless/src/submitCmrOrderingOrder/handler.js

```diff
@@ -1,21 +1,22 @@
 import 'array-foreach-async'
-
 import axios from 'axios'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
I think this could be replaced by 
```js
import { randomUUID } from 'node:crypto'

const requestId = randomUUID()
```

Reply:
It does look like it works, I'd like to write a ticket to replace `uuid` across the project so we can remain consistent in the code